client/packages/system/src/IndicatorsDemographics/DetailView/IndicatorsDemographics.tsx

```diff
@@ -136,27 +102,28 @@ const IndicatorsDemographicsComponent = () => {
       ...patch,
       value: patch.value ?? 0,
     };
-    setHeaderDraft({ ...getHeaderDraft(), [patch.id]: updatedPatch });
+    setHeaderDraft({
+      ...(headerDraft as HeaderData),
```

Comment:
Looks like a lot of them aren't your changes, but I'm a bit worried about the number of `as Type`s that we have going on in this file... maybe something else to see if we can fix in the cleanup issue 👀